docs/release/itb-testing.md

```diff
@@ -25,50 +28,85 @@ The commands can be run as-is from within the `osgitb` directory from git.
         :::console
         ansible current -i inventory -f 20 -o -m command -a 'cat /etc/redhat-release'
 
-2. Check HTCondor versions for all HTCondor hosts:
+1. Check the date and time on all hosts (in case NTP stops working):
 
         :::console
-        ansible condor -i inventory -f 20 -o -m command -a 'rpm -q condor'
+        ansible current -i inventory -f 20 -o -m command -a 'date'
+
+1. Check software versions for certain hosts (e.g., for the `condor` package on hosts in the `worksers` group):
```

Comment:
worksers typo?

Reply:
Good catch! Fixing… and I ran a spelling check on the whole document this time.